input/templates/partials/catalog/quickview.hbs

```diff
@@ -43,7 +44,7 @@
             </div>
             <div class="quickview-wishlist-row">
               <ul class="list-inline">
-                <li>
+                <li {{#if wishlist}}class="hidden"{{/if}}>
```

Comment:
I guess it was working before fine, but I'm surprised about that. Shouldn't it be `{{#unless wishlist}}class="hidden"{{/unless}}`? At least semantically :D